### PR TITLE
Updating prerelease docs to include SE

### LINKED ISSUE #485: Add Testing Prerelease Content for SE
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [x] New content (guide wanted)
- [ ] Update (Add missing or refresh existing content)

## Expected Behavior
Have content similar to https://docs.sensu.io/sensu-core/1.4/installation/testing-prereleases/

## Current Behavior
We don't currently have info on testing SE prereleases.

### PR DESCRIPTION
**Pull request by asachs01**
## Description
Adds SE testing repos to testing pre-release doc.

## Motivation and Context
Fixes #485 

## How Has This Been Tested?
`yarn server`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New doc/guide
- [ ] Fixing errata
- [x] Update (Add missing or refresh existing content)

## Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] All tests have passed.

### CODE REVIEW

**Comment by cwjohnston** [comment]
@asachs01 Instead of adding this here, I think we should figure out how to make this content work in the Sensu Enterprise section.

**Comment by asachs01** [comment]
@cwjohnston this has now been split out to SE. There's only the singular article in there now, but I anticipate we'll have more. I still need to confirm that the URLs are correct for the repos.

**Comment by asachs01** [comment]
Also addresses #501

File: content/sensu-enterprise/2.6/installation/testing-prereleases.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Testing and using a Sensu prerelease is straightforward, it simply
+involves using a different package repository. You should only be
+using prereleases if you already have experience installing and
+managing a Sensu installation. Below are the per-platform prerelease
+repository installation instructions. We are slowly adding prerelease
+repositories for other platforms.
+
+_NOTE: The Sensu prerelease repository instructions replace the
+existing Sensu repository configuration files, you will need to revert
+these changes in order to return to using stable releases._
```
Comment (rgeniesse) [inline content/sensu-enterprise/2.6/installation/testing-prereleases.md]: Maybe have this as two sentences or change it to `...configuration files, so you will need to `?

File: content/sensu-enterprise/2.6/installation/testing-prereleases.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+1. Create the YUM repository configuration file for the Sensu Core repository at
+   `/etc/yum.repos.d/sensu.repo`:
+   {{< highlight shell >}}
+echo '[sensu]
+name=sensu
+baseurl=https://sensu.global.ssl.fastly.net/yum-unstable/$releasever/$basearch/
+gpgcheck=0
+enabled=1' | sudo tee /etc/yum.repos.d/sensu.repo{{< /highlight >}}
+
+2. Install Sensu
```
Comment (rgeniesse) [inline content/sensu-enterprise/2.6/installation/testing-prereleases.md]: Add a `:` here to stay consistent with the end of the other steps on this page.

File: content/sensu-enterprise/3.0/installation/testing-prereleases.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+echo "[sensu-enterprise]
+name=sensu-enterprise
+baseurl=http://$SE_USER:$SE_PASS@enterprise.sensuapp.com/yum-unstable/noarch/
+gpgcheck=0
+enabled=1" | sudo tee /etc/yum.repos.d/sensu-enterprise.repo{{< /highlight >}}
+
+2. Install Sensu
+{{< highlight shell >}}
+sudo yum update sensu{{< /highlight >}}
+## Reporting issues
```
Comment (rgeniesse) [inline content/sensu-enterprise/3.0/installation/testing-prereleases.md]: Need a new line above here, as this causes the `Reporting issues` to indent funny. The other versions look ok.

**Review by cwjohnston — DISMISSED** [review]
I think Richie’s suggestions merit some changes here before we merge.

File: content/sensu-enterprise/2.6/installation/testing-prereleases.md (hunk 1 above)
Comment (asachs01) [reply content/sensu-enterprise/2.6/installation/testing-prereleases.md]: :+1:

File: content/sensu-enterprise/2.6/installation/testing-prereleases.md (hunk 2 above)
Comment (asachs01) [reply content/sensu-enterprise/2.6/installation/testing-prereleases.md]: :+1:

**Comment by asachs01** [comment]
@rgeniesse @cwjohnston the changes have been pushed. Doublecheck me again?

**Review by rgeniesse — APPROVED** [review]
Changes look good. 👍

**Pull request merged**